sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -86,19 +84,11 @@
         return isAuthorized(ManagementConstants.OPERATION_CANCEL_SCHEDULED_MESSAGE)
             .then(createChannel.flatMap(channel -> {
                 final Message requestMessage = createManagementMessage(
-                    ManagementConstants.OPERATION_CANCEL_SCHEDULED_MESSAGE, channel.getReceiveLinkName());
+                    ManagementConstants.OPERATION_CANCEL_SCHEDULED_MESSAGE, null);
 
                 requestMessage.setBody(new AmqpValue(Collections.singletonMap(ManagementConstants.SEQUENCE_NUMBERS,
                     new Long[]{sequenceNumber})));
                 return channel.sendWithAck(requestMessage);
-            }).map(responseMessage -> {
-                int statusCode = RequestResponseUtils.getResponseStatusCode(responseMessage);
-
-                if (statusCode == AmqpResponseCode.OK.getValue()) {
-                    return Mono.empty();
```

Comment:
How are we checking the error condition and throw error when service did not send OK status code ?

Reply:
Resolved